src/js/constants.js

```diff
@@ -1,4 +1,9 @@
 export const STUY_SPEC_API = 'https://api.stuyspec.xyz';
 export const STUY_SPEC_API_HEADER = {
   'X-Key-Inflection': 'camel',
-};
\ No newline at end of file
+};
+
+export const ROLE_SLUG_OF_MEDIA_TYPE = {
```

Comment:
It's an object to turn media types (i.e. illustration, photograph) into role slugs (i.e. illustrators, photographers). This conversion is useful for getting the slug of the media type so a featured media caption can link to a creator. Look in ArticleFeaturedMedia:35.